gremlin-go/example/go.mod

```diff
@@ -0,0 +1,29 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+module example
+
+go 1.17
+
+require github.com/lyndonbauto/tinkerpop/gremlin-go/v3 v3.5.3-rc12
```

Comment:
Needs to be apache/tinkerpop

Reply:
updated in https://github.com/lyndonbauto/tinkerpop/pull/94/commits/e2e86986e299552753eab71559aa9cb7fd18643d